mod/consensus-types/pkg/types/state.go

```diff
@@ -378,7 +378,7 @@ func (st *BeaconState[
 	}
 	subIndx = hh.Index()
 	for _, i := range st.Slashings {
-		hh.AppendUint64(i)
+		hh.AppendUint64(uint64(i))
```

Comment:
q: should I use `i.Unwrap()` here instead? It seems the principled way, but it makes a copy, while casting should not.  We do use the uint64 cast for TotalSlashing as well (line 392)